app/models/email_history.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class EmailHistory < ApplicationRecord
+  EARLY_ALLOCATION_CREATE = 'create_early_allocation'
```

Comment:
I think we're going to need two different events: one for each possible EA outcome which triggers an email.

The prototype [case history page](https://hmpps-moic-staging.herokuapp.com/ea-case-activity) differentiates between 'eligible' and 'discretionary' emails sent to the LDU:

<img width="938" alt="Screenshot 2020-11-30 at 16 34 33" src="https://user-images.githubusercontent.com/7735945/100636833-0f1af500-332a-11eb-9951-8e557681a72d.png">

Note the difference in title between:

- **Early allocation decision requested** (for a discretionary outcome)
- **Early allocation assessment form sent** (for an eligible outcome)

Or were you thinking that the type of email sent would be inferred from the outcome on the Early Allocation record?

---

### Naming

In terms of naming, I think it might be better to use `_REFERRAL` rather than `_CREATE` in the name – after all, the reason for this email is _referring it to the LDU_, not just _creating_ the early allocation record. But that's just my opinion – naming is hard.